pages/index.js

```diff
@@ -33,14 +33,9 @@ export default function Home(props) {
             dataCyButton="ExperimentsButton"
           >
             <h1>{t("experimentsAndExplorationTitle")}</h1>
```

Comment:
For now can we put some margin underneath this heading? I think something like mb-4 should be fine.